Add new events `pm:dragenable`, `pm:dragdisable`, `pm:change` (#1112)
geoman-io/leaflet-geoman: README.md

| pm:snapdrag       | `e`    | Fired during a marker move/drag. Payload includes info about involved layers and snapping calculation.                | `shape`, `distance`, `layer` = `workingLayer`, `marker`, `layerInteractedWith`, `segment`, `snapLatLng` |
| pm:snap           | `e`    | Fired when a vertex is snapped. Payload is the same as in `snapdrag`.                                                 | `shape`, `distance`, `layer` = `workingLayer`, `marker`, `layerInteractedWith`, `segment`, `snapLatLng` |
| pm:unsnap         | `e`    | Fired when a vertex is unsnapped. Payload is the same as in `snapdrag`.                                               | `shape`, `distance`, `layer` = `workingLayer`, `marker`, `layerInteractedWith`, `segment`, `snapLatLng` |
| pm:centerplaced   | `e`    | Called when the center of a circle is placed/moved.                                                                   | `shape`, `workingLayer`, `latlng`                                                                       |

> | pm:positionchange | `e`    | Fired when position / coordinates of a layer changed.                                                                 | `layer`, `latlng`, `shape`                                                                              |

Reviewer: ```suggestion
| pm:change | `e`    | Fired coordinates of the layer changed.                                                                 | `layer`, `latlngs`, `shape`                                                                              |
```
